lib/octicons_node/README.md

```diff
@@ -1,24 +1,24 @@
 # @primer/octicons
 
 
-[![npm version](https://img.shields.io/npm/v/@primer/octicons.svg)](https://www.npmjs.org/package/@primer/octicons)
+[![npm version](https://img.shields.io/npm/v/@openproject/octicons-angular.svg)](https://www.npmjs.org/package/@openproject/octicons-angular)
 
 ## Install
 
-This package is distributed with [npm][npm]. After [installing npm][install-npm], you can install `@primer/octicons` with this command:
+This package is distributed with [npm][npm]. After [installing npm][install-npm], you can install `@openproject/octicons-angular` with this command:
 
 ```shell
-npm install @primer/octicons
+npm install @openproject/octicons-angular
 ```
 
 ## Usage
 
-For all the usages, we recommend using the CSS located in [`build/build.css`](https://unpkg.com/@primer/octicons/build/build.css). This is some simple CSS to normalize the icons and inherit colors.
+For all the usages, we recommend using the CSS located in [`build/build.css`](https://unpkg.com/@openproject/octicons-angular/build/build.css). This is some simple CSS to normalize the icons and inherit colors.
```

Comment:
Same here, the `build/build.css` does not exists (and the URL is showing nothing). Should we add that?

Reply:
This comes from the `octicons_node` package. All it includes is the contents of `lib/octicons_node/index.scss`:

```
.octicon {
  display: inline-block;
  vertical-align: text-top;
  fill: currentColor;
  overflow: visible;
}
```

Not sure how to handle this. We could publish the node package after all, or just do the same sass build step in the angular package and publish the css file there as well. wdyt?